Plots/Contours/NCL_eof_1_1.py

```diff
@@ -0,0 +1,326 @@
+"""
+NCL_eof_1_1.py
+===============
+Calculate EOFs of the Sea Level Pressure over the North Atlantic.
+
+This script illustrates the following concepts:
+  - Calculating EOFs
+  - Drawing a time series plot
+  - Using coordinate subscripting to read a specified geographical region
+  - Rearranging longitude data to span -180 to 180
+  - Calculating symmetric contour intervals
+  - Drawing filled bars above and below a given reference line
+  - Drawing subtitles at the top of a plot
+  - Reordering an array
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/eof_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/eof_1_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/eof_1_2_lg.png
+
+Note (1):
+    So-called original NCL plot "eof_1_2_lg.png" given in the above URL is likely not identical to what the given NCL original script generates. When the given NCL script is run, it generates a plot with identical data to that is plotted by this Python script.
+
+Note (2):
+    This script includes many optional diagnostic print statements to provide information about data slicing. To activate such print statements, the parameter "debug" should be set to True.
+"""
+
+
+###############################################################################
+# Import packages:
+import xarray as xr
+from math import atan
+import numpy as np
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+from geocat.viz import cmaps as gvcmaps
+from geocat.comp import eofunc, eofunc_ts
+
+import matplotlib.pyplot as plt
+
+import cartopy.crs as ccrs
+
+###############################################################################
+# User defined parameters and a convenience function:
+
+# In order to specify region of the globe, time span, etc.
+latS = 25.
+latN = 80.
+lonL = -70.
+lonR = 40.
+
+yearStart = 1979
+yearEnd = 2003
+
+neof = 3   # number of EOFs
+optETS = False
```

Comment:
what is optETS?

Reply:
It should be removed; I think it may have been used at some point in the NCL script, but it is no longer being used.